applications/machine_learning/README.rst

```diff
@@ -275,6 +275,28 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or power off there are not in use.
+
+* In the suspended state, the device maintains the active connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and advertising is stopped.
+
+Any button press can wake up the device.
+
+For the Thingy:53 sensors supports active power management.
```

Comment:
It would be valuable to add a link to active power management from sensor manager here. I could add a label for it here https://github.com/nrfconnect/sdk-nrf/pull/6329/commits/b30dcbb1a6f5fc7de3c1d0f2c010b2e97c899702#diff-1e404cbcd69774fbac51b9195758eabdfb04431d39c7b2a3b9822ef145270f1aR304

Reply:
Discussed offline. Let's skip this for now.

Reply:
Agreed offline to ignore this proposal.